app/controllers/services_controller.rb

```diff
@@ -14,6 +14,7 @@ def edit
 
   def update
     if @bibliography_service.update(update_params)
+      @exhibit.sync_bibliography if @bibliography_service.api_settings_changed?
```

Comment:
Yeah I came to a similar conclusion and felt like it would need to jump through some hoops. Maybe another model? But ... /shrug